deploy/helm/sumologic/templates/clusterrole.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: {{ template "sumologic.metadata.name.roles.clusterrole" . }}
+  namespace: {{ template "sumologic.namespace"  . }}
```

Comment:
`ClusterRole`s are not namespaced
```suggestion
```

Reply:
@astencel-sumo This has been now removed. Earlier goldentest needed it so we added it here but ideally yes we don't need namsepace for non-namespaced resource.